bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -112,6 +112,14 @@ pub struct QueryCosts {
     )]
     pub dry_run: usize,
 
+    /// Query costs for generating execution trace for a block.®
```

Comment:
```suggestion
    /// Query costs for generating execution trace for a block
```
I feel like I'm ruining some fun here. But I'm not sure it belongs

Reply:
Hahahah it was a typo, but keeping it would have been funny. Removed in fc097bf56e.